lib/src/over_react_redux/devtools/middleware.dart

```diff
@@ -193,3 +193,20 @@ class _OverReactReduxDevToolsMiddleware extends MiddlewareClass {
 /// );
 /// ```
 final MiddlewareClass overReactReduxDevToolsMiddleware = _OverReactReduxDevToolsMiddleware();
```

Comment:
I almost wonder if we should deprecate this in favor of `overReactReduxDevToolsMiddlewareFactory`, to encourage people to add names. Thoughts?

Reply:
Maybe eventually

Technically this approach of a middleware factory is not the method that the official redux-devtools uses. Their implementation looks a bit more like:

```js
const composeEnhancers = composeWithDevTools(options);
const store = createStore(
  reducer,
  /* preloadedState, */ composeEnhancers(
    applyMiddleware(...middleware)
    // other store enhancers if any
  )
);
```

I wasn't sure of the core reasoning behind this, so I felt a bit more comfortable "trying this out" for a bit before we consider it the de-facto way to initialize a dev tools middleware.

If you don't think that's necessary, I'm happy to deprecate and encourage the factory approach

Reply:
That's fine by me 👍